src/test/resources/feature/3-destinatario/personaGiuridica/downloadFile/downloadAOTPG.feature

```diff
@@ -0,0 +1,44 @@
+Feature: persona giuridica scarica attestazione opponibile
+
+  @TestSuite
+  @TA_PG_DownloadFileAOTPresaInCarico
+  @DownloadFilePG
+  @PG
+
+  Scenario: PN-10432 - Persona giuridica scarica Attestazione opponibile a terzi: notifica presa in carico
+    Given Login Page mittente "mittente_1" viene visualizzata
+    When Login con mittente "mittente_1"
+    And Si clicca sul bottone test
+    And Si clicca bottone accetta cookies
+    Then Home page mittente viene visualizzata correttamente
+    And Nella pagina Piattaforma Notifiche cliccare sul bottone Invia una nuova notifica
+    And Si visualizza correttamente la pagina Piattaforma Notifiche section Informazioni preliminari
+    Then Nella section Informazioni preliminari si inseriscono i dati della notifica senza gruppo
```

Comment:
create noticication with api and delete testsuite

Reply:
cant create with api, because method for creation of notifica done by Gabri, uses as mittente only Palermo. In our case we need Viggiu

Reply:
cant create notification with api, because mittente is comune Viggiu